index.js

```diff
@@ -75,6 +93,29 @@ const registerBearer = function(server, options) {
 
 /**
  * @param {Object} server
+ * @param {Plan3KeyOptions} options
+ * @return {Promise}
+ */
+const registerPlan3KeyStrategy = function(server, options) {
```

Comment:
please validate here, that options has a 'tokens' field - same for Bearer. I just got hit by that. Thanks 🙇